napari/layers/base/base.py

```diff
@@ -1405,7 +1434,7 @@ def _get_ray_intersections(
         world : bool
             True if the provided coordinates are in world coordinates.
             Default value is True.
-        bounding_box : np.ndarray
+        bounding_box: np.ndarray
```

Comment:
Are these colons supposed to always have a space on both sides or is this ok?

Reply:
Yeah, if there is no space it's missparsed by numpydoc.
```suggestion
        bounding_box : np.ndarray
```

Reply:
Good catch. Thanks!